lib/Doctrine/Migrations/Configuration/Connection/Loader/ConnectionHelperLoader.php

```diff
@@ -14,30 +14,29 @@
  *
  * @internal
  */
-class ConnectionHelperLoader implements ConnectionLoaderInterface
+final class ConnectionHelperLoader implements ConnectionLoaderInterface
 {
     /** @var string */
     private $helperName;
 
     /** @var  HelperSet */
     private $helperSet;
 
-    public function __construct(?HelperSet $helperSet = null, string $helperName)
+    /** @var ConnectionLoaderInterface */
+    private $fallback;
+
+    public function __construct(?HelperSet $helperSet = null, string $helperName, ConnectionLoaderInterface $fallback)
```

Comment:
Can we move parameters with a default value to the end of the parameter list, or better even, drop the default value?

Also see 

* https://github.com/localheinz/phpstan-rules#methodsnoparameterwithnullabletypedeclarationrule
* https://github.com/localheinz/phpstan-rules#methodsnoparameterwithnulldefaultvaluerule